src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -2213,7 +2214,7 @@ namespace OpenRCT2::Ui::Windows
         {
             // If the scenery tool is active, we do not display our tiles as it
             // will conflict with larger scenery objects selecting tiles
-            if (isToolActive(WindowClass::Scenery))
+            if (gMapSelectFlags & MAP_SELECT_FLAG_ENABLE)
```

Comment:
Not sure if that change is correct, need some testing still.

Reply:
Oh, this is responsible for keeping the tiles of the provisional piece marked with a yellow border.  
Didn't find any negative interference with the footpath tool. (When playing a provisional footpath in one of the tiles of the provisional track piece, the footpath tile gets a white border while the others remain yellow. That seams reasonable.) So I think this can stay as it was. Reverting all my changes in this line.